roadart/bin/videos_to_training.dart

```diff
@@ -0,0 +1,138 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:crypto/crypto.dart';
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:googleapis/drive/v3.dart' as drive;
+import 'package:http/http.dart' as http;
+import 'package:process_run/process_run.dart';
+import 'package:yaml/yaml.dart';
+
+class _Runner {
+  static const String kRelativeOauthSecretPath = 'secret/oauth_secret.json';
+  static const String kRelativeAccessTokenPath = 'secret/access_token.json';
+
+  final String yamlPath;
+
+  _Runner(this.yamlPath) {
+    _rootPath = File(yamlPath).parent.parent.path;
```

Comment:
If this is the case, could this program take root path as a parameter? Or could it take both root path and yaml path as parameters?

Reply:
Sure, please see the new commit.